src/pages/_app.jsx

```diff
@@ -1,95 +1,80 @@
 /* eslint-disable react/jsx-props-no-spreading */
-import React, { useEffect, useState } from 'react';
+import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
 import { DefaultSeo } from 'next-seo';
-import Head from 'next/head';
 import PropTypes from 'prop-types';
 import Router, { useRouter } from 'next/router';
 import NProgress from 'nprogress';
-import useSWR from 'swr';
-
+import Amplify from 'aws-amplify';
+import _ from 'lodash';
 import ContentWrapper from '../components/ContentWrapper';
-import PreloadContent from '../components/PreloadContent';
 import NotFoundPage from './404';
 import Error from './_error';
-import wrapper from '../redux/store';
-import getFromApiExpectOK from '../utils/getFromApiExpectOK';
+import { wrapper } from '../redux/store';
 import '../../assets/self-styles.less';
 import '../../assets/nprogress.css';
+import configure from '../utils/amplify-config';
+import CustomError from '../utils/customError';
 
-import { isBrowser, getCurrentEnvironment } from '../utils/environment';
-import setupAmplify from '../utils/setupAmplify';
+// import { isBrowser } from '../utils/environment';
+// TODO: this needs to be refactored after auth works properly
+//
+// import { isBrowser, getCurrentEnvironment } from '../utils/environment';
+// import setupAmplify from '../utils/setupAmplify';
 
 Router.events.on('routeChangeStart', () => NProgress.start());
 Router.events.on('routeChangeComplete', () => NProgress.done());
 Router.events.on('routeChangeError', () => NProgress.done());
 
-const WrappedApp = ({ Component, pageProps, req }) => {
+const WrappedApp = ({ Component, pageProps }) => {
+  const { httpError } = pageProps;
   const router = useRouter();
-  const [experimentId, setExperimentId] = useState(undefined);
 
-  console.log(req, pageProps);
+  const { auth } = useSelector((state) => state.networkResources);
```

Comment:
Authentication config such as userPool and identityPool settings are fetched from Redux.